### DISCUSSION TITLE
ST7735 and ST7789 in the same time

### DISCUSSION THREAD

**Discussion by jerecraft54** [Bodmer/TFT_eSPI#2350] [General]
hello everybody,

i would like to know if we can use tst-espi library with 2 screens but one ST7735 and one ST7789 in the same time?
with 2 screens ST7735, it's ok but i want to try with differents screens.
What would be the user_setup.h?

thank you for your help

**Comment by Bodmer** [comment]
The library does not support the simultaneous use of two different display types. Due to the rarity of this need there are no plans to add this feature.

**Comment by jerecraft54** [comment]
Thank you for the reply
I use 2 st7735 simultaneous and the library works fine
Good job for the library

Envoyé à partir de Outlook pour iOS<https://aka.ms/o0ukef>
________________________________
De : Bodmer ***@***.***>
Envoyé : Wednesday, January 25, 2023 1:28:18 PM
À : Bodmer/TFT_eSPI ***@***.***>
Cc : jerecraft54 ***@***.***>; Author ***@***.***>
Objet : Re: [Bodmer/TFT_eSPI] ST7735 and ST7789 in the same time (Discussion #2350)


The library does not support the simultaneous use of two different display types. Due to the rarity of this need there are no plans to add this feature.
